src/Computer.php

```diff
@@ -140,6 +140,7 @@ public function defineTabs($options = [])
          ->addStandardTab('Item_Ticket', $ong, $options)
          ->addStandardTab('Item_Problem', $ong, $options)
          ->addStandardTab('Change_Item', $ong, $options)
+         ->addStandardTab('Item_Project', $ong, $options)
```

Comment:
have you / can you check that all the following objects have the `Project` tab

```php
$CFG_GLPI["project_asset_types"]          = ['Computer', 'Monitor', 'NetworkEquipment',
    'Peripheral', 'Phone', 'Printer', 'Software',
    'DeviceMotherboard', 'DeviceProcessor', 'DeviceMemory',
    'DeviceHardDrive', 'DeviceNetworkCard', 'DeviceDrive',
    'DeviceControl', 'DeviceGraphicCard', 'DeviceSoundCard',
    'DevicePci', 'DeviceCase', 'DevicePowerSupply', 'DeviceGeneric',
    'DeviceBattery', 'DeviceFirmware', 'DeviceCamera',
    'Certificate', 'Appliance'
];
```

Reply:
It can be done in a PHPUnit test.

```php
foreach ($CFG_GLPI["project_asset_types"] as $itemtype) {
    $item = new $itemtype();
    $tabs = $item->defineTabs();
    // assert tabs contains an entry for Item_Project
}